packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactClippingViewGroupHelper.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+ package com.facebook.react.uimanager
+
+ import android.graphics.Rect
+ import android.view.View
+ import com.facebook.infer.annotation.Nullsafe
+ import javax.annotation.concurrent.NotThreadSafe
+ 
+ /**
+  * Provides implementation of common tasks for view and it's view manager supporting property
+  * [removeClippedSubviews].
+  */
+ @Nullsafe(Nullsafe.Mode.LOCAL)
+ @NotThreadSafe
+ public object ReactClippingViewGroupHelper {
+ 
+     public const val PROP_REMOVE_CLIPPED_SUBVIEWS: String = "removeClippedSubviews"
+ 
+     private val sHelperRect: Rect = Rect()
```

Comment:
Let's rename this to avoid using Hungarian notation:

```suggestion
     private val helperRect: Rect = Rect()
```

Reply:
Changed